src/sentry/api/endpoints/organization_traces.py

```diff
@@ -718,7 +726,6 @@ def get_traces_matching_span_conditions_query_indexed(
                 snuba_params=snuba_params,
                 query=None,
                 selected_columns=["trace", timestamp_column],
-                orderby=orderby,
```

Comment:
Why are we removing sort functionality on the indexed spans dataset?

Reply:
This timed out most of the time so was disabled. Now it's only supported on the eap spans.